fastlane/spec/helper/podspec_helper_spec.rb

```diff
@@ -0,0 +1,74 @@
+require "stringio"
+
+describe Fastlane::Actions do
+  describe "#podspechelper" do
+    before do
+      @version_podspec_file = Fastlane::Helper::PodspecHelper.new
+    end
+
+    it "raises an exception when an incorrect path is given" do
+      expect do
+        Fastlane::Helper::PodspecHelper.new('invalid_podspec')
+      end.to raise_error("Could not find podspec file at path 'invalid_podspec'")
+    end
+
+    it "raises an exception when there is no version in podspec" do
+      expect do
+        Fastlane::Helper::PodspecHelper.new.parse("")
+      end.to raise_error("Could not find version in podspec content ''")
+    end
+
+    it "raises an exception when the version is commented-out in podspec" do
+      test_content = '# s.version = "1.3.2"'
+      expect do
+        Fastlane::Helper::PodspecHelper.new.parse(test_content)
+      end.to raise_error("Could not find version in podspec content '#{test_content}'")
+    end
+
+    context "when semantic version" do
+      it "returns the current version once parsed" do
+        test_content = 'spec.version = "1.3.2"'
+        result = @version_podspec_file.parse(test_content)
+        expect(result).to eq('1.3.2')
+        expect(@version_podspec_file.version_value).to eq('1.3.2')
+        expect(@version_podspec_file.version_match[:major]).to eq('1')
+        expect(@version_podspec_file.version_match[:minor]).to eq('3')
+        expect(@version_podspec_file.version_match[:patch]).to eq('2')
+      end
+
+      it "returns the current version once parsed with appendix" do
+        test_content = 'spec.version = "1.3.2.4"'
```

Comment:
I think adding a test for the behavior of `spec.version = "1.2.3.4.5"` returning `.4.5` as the `:appendix` would be good, just to guarantee that the old behavior is, in-fact, preserved in to the future.
I don't feel super strongly though, so if you have a reason why that's a dumb idea, feel free to ignore and 🎈'it